README.md

```diff
@@ -51,11 +51,26 @@ Windows Application Driver can run remotely on any Windows 10 machine with `WinA
    - Choose specific local port (4723 is WinAppDriver standard)
    - **Action** -> **Allow the connection**
    - **Profile** -> select all
-   - **Name** -> optional, choose name for rule (e.g. WinAppDriver remote)
+   - **Name** -> optional, choose name for rule (e.g. WinAppDriver remote).
+   
+   Below command when run in admin command prompt gives same result
+   ```
+   netsh advfirewall firewall add rule name="Windows application driver" dir=in action=allow protocol=TCP localport=4723
```

Comment:
Did you mean `"WinAppDriver remote"` instead of `"Windows application driver"`. Otherwise, we can change the e.g. section of the optional **Name** entry to provide more clarity.

Reply:
I updated the readme as per your suggestion...[Here](https://github.com/Microsoft/WinAppDriver/pull/366) is updated ull request